### DISCUSSION TITLE
Question about non-VS Code support and board support

### DISCUSSION THREAD

**Discussion by vincentchov** [BradenM/micropy-cli#302] [Q&A] [answered]
Is the Micropy CLI compatible with other text editors like NeoVim? Also, are the stubs compatible with the ESP8266? Thanks!

Also, I made an issue (#301) for visibility.

**Comment by BradenM** [comment, marked as answer]
Hi @vincentchov,

Strictly speaking, `micropy-cli` provides standard [PEP 484](https://peps.python.org/pep-0484/#storing-and-distributing-stub-files) stub files that are compatible with any python language server protocol implementation.

One would only need to configure (or be able to configure) said LSP implementation to look for the stubs located in a project's `.micropy` folder. How this is done will vary of course depending on what IDE/editor you are using.

Currently, `micropy-cli` only automates this process for vs-code, but there is nothing preventing support of other IDEs other than time :).

Hopefully this answers your question, and sorry for taking a while to get to it. Thanks!
